packages/util/package.json

```diff
@@ -19,20 +25,7 @@
     "test": "vitest run"
   },
   "devDependencies": {
-    "@babel/core": "^7.20.12",
-    "@babel/preset-env": "^7.20.2",
-    "@cloudinary-util/tsconfig": "workspace:*",
-    "@colbyfayock/semantic-release-pnpm": "^1.0.7",
-    "@semantic-release/changelog": "^6.0.2",
-    "@semantic-release/git": "^10.0.1",
-    "@types/node": "^17.0.12",
-    "eslint": "^7.32.0",
-    "eslint-config-cloudinary-util": "workspace:*",
-    "semantic-release": "^20.1.0",
-    "semantic-release-monorepo": "^7.0.5",
-    "tsup": "^5.10.1",
-    "typescript": "^4.5.3",
-    "vitest": "^1.2.0"
+    "@types/node": "^17.0.12"
   },
```

Comment:
question: should all these packages be removed? Like `vitest` for example for testing?

Reply:
These weren't removed, I just moved them to `devDependencies` at the top-level of the monorepo.

That way, the versions aren't maintained in multiple places and since they're only used in development across packages, it's generally easier to maintain dev deps this way in a monorepo.

Reply:
Very nice improvement!